bukuserver/views.py

```diff
@@ -525,19 +517,20 @@ def top_most_common_func(query, value, index):
             bs_filters.TagBaseFilter(name, filter_type=FilterType.NOT_EQUAL),
             bs_filters.TagBaseFilter(name, filter_type=FilterType.IN_LIST),
             bs_filters.TagBaseFilter(name, filter_type=FilterType.NOT_IN_LIST),
-            bs_filters.TagBaseFilter(name, filter_type=FilterType.CONTAINS),
-            bs_filters.TagBaseFilter(name, filter_type=FilterType.NOT_CONTAINS),
```

Comment:
…Just noticed these were added to the `usage_count` filter group as well (even though they expect strings and not numbers :sweat_smile:)